src/components/MdMenu/MdMenuContent.vue

```diff
@@ -206,10 +203,30 @@
               this.MdMenu.active = false
               this.MdMenu.bodyClickObserver.destroy()
               this.MdMenu.windowResizeObserver.destroy()
+              this.destroyKeyDownListener()
             }
           })
         }
       },
+      createKeydownListener () {
+        window.addEventListener('keydown', this.keyNavigation)
+      },
+      destroyKeyDownListener () {
+        window.removeEventListener('keydown', this.keyNavigation)
+      },
+      keyNavigation (event) {
+        switch (event.key) {
```

Comment:
If the page has scrollbars, the event is not working as it should be. Maybe adding a `event.preventDefault()` can prevent window scrollbars from scroll.